server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -13,6 +13,8 @@
 
 class ScanpyEngine(CXGDriver):
 
+    __version__ = "0.0.1"
```

Comment:
Is there a compelling reason to version each individual `CXGDriver`?

Reply:
Unfortunately, yes. If someone is running analysis through cellxgene then reproducibility is dependent on the version of the engine they used.

Reply:
Shouldn't cellxgene itself just be versioned?

Reply:
Maybe? But the engines are so independent, someone using the (future) EM2 engine wouldn't care about updates to the scanpy engine. That was my reasoning as far as versioning at the scanpy engine level.

Reply:
I guess, but it's two separate version numbers to keep track of now (for cellxgene and for the engines).

Reply:
Definitely would prefer a solution that has me updating versions less. You are suggesting we only version cellxgene instead of either the driver or the engines, right?

Reply:
Exactly.

Reply:
Ok, moved it to the cxg version. I'll just bump that faster than I ordinarily would.